app/controllers/application_controller.rb

```diff
@@ -7,4 +7,8 @@ def ssl_configured?
     !Rails.env.production?
     !Rails.env.test?
   end
+
+  def after_sign_in_path_for(_resource)
```

Comment:
You take in a `_resource` but don't use the argument. Is this supposed to be that way?